classes/import-export.php

```diff
@@ -568,7 +587,7 @@ public function do_csv_preview() {
 			// Include CSV library.
 			include_once( SIMPLEMAP_PATH . '/classes/parsecsv.lib.php' );
 
-			$file_location = WP_PLUGIN_DIR . '/sm-temp-csv-' . $blog_id . '.csv';
+			$file_location = $this->get_import_path( $blog_id . '.csv' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->